docs/restrict-cross-namespace-associations.asciidoc

```diff
@@ -0,0 +1,97 @@
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/restrict-cross-namespace-associations.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-restrict-cross-namespace-associations"]
+=== Restrict cross namespace resource associations
+
+By default, associations can be created between any resources as long as they are deployed in some namespaces managed by the same ECK instance.
+
+This is true, for example, when using the `elasticsearchRef` field in order to automatically configure and establish a trust relationship between Kibana and Elasticsearch. If a user can create a Kibana instance in a namespace managed by ECK then this Kibana instance can be associated with any Elasticsearch cluster deployed in another namespace managed by the same ECK instance.
+
+This section describes how you can restrict the associations that can be created between resources managed by ECK.
+
+[float]
+==== Enabling cross namespace association restriction
+
+This feature is disabled by default. To enable it start the operator with the `--enforce-rbac-on-refs` flag.
+
+NOTE: This feature only provides an access control for resources deployed across two different namespaces. Associations can still be freely created between resources deployed in a same namespace.
```

Comment:
```suggestion
NOTE: This feature only provides an access control for resources deployed across two different namespaces. You can still create associations between resources deployed in a same namespace.
```

Reply:
This needs to be: "in **the** same namespace. "